Examples/destination_plugins/FlurryDestination.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  FlurryDestination.swift
+//  SegmentUIKitExample
+//
+//  Created by Brandon Sneed on 4/9/21.
+//
+
+// NOTE: You can see this plugin in use in the SwiftUIKitExample application.
+//
+// This plugin is NOT SUPPORTED by Segment.  It is here merely as an example,
+// and for your convenience should you find it useful.  Please contact Flurry
+// about providing full support and publishing if desired.
+//
+// Flurry SPM package can be found here: https://github.com/flurry/FlurrySwiftPackage
+
+// MIT License
+//
+// Copyright (c) 2021 Segment
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Foundation
+import Segment
+/**
+ An implmentation of the Flurry Analytics device mode destination as a plugin.
+ 
+ NOTE: This is currently commented out due to lack of arm64 support in the flurry SDK via swift package manager.
+ See here: https://github.com/flurry/FlurrySwiftPackage/issues/3
+ */
+
+/*
+import FlurryAnalytics
```

Comment:
This worked for me, we sure we need to comment it out? I just need to do a clean build each time on Xcode 12.5. 12.4 it works without having to do that /eyeroll

Reply:
It works for you because you don't have an M1 Mac.  That's the issue.

Reply:
Interesting, because I was getting that "warning" which resulted in a build error until then. But I'm with you. I think we're going to run into that on almost all of these.